src/pages/organizations/[organizationid].js

```diff
@@ -190,14 +194,14 @@ export default function Organization(props) {
               </Typography>
               <Box sx={{ mt: 2 }}>
                 <Info
-                  iconURI={calendarIcon}
+                  iconURI={CalendarIcon}
                   info={`Organization since ${moment().format(
                     'MMMM DD, YYYY',
                   )}`}
                 />
               </Box>
               <Box sx={{ mt: 2 }}>
-                <Info iconURI={locationIcon} info="Shirimatunda, Tanzania" />
+                <Info iconURI={LocationIcon} info="Shirimatunda, Tanzania" />
```

Comment:
@jeremydthomas  for example, here, you put into the SVG to replace the `src`, but in the component:

```
    <Box
      sx={{
        color: 'text.text2',
        display: 'flex',
        alignItems: 'center',
        gap: 3,
        '& img': {
          filter: 'opacity(0.5)',
          maxWidth: 16,
          maxHeight: 16,
        },
      }}
    >
      <SvgIcon component={iconURI} inheritViewBox alt="join time" />
      <Typography variant="h6">{info}</Typography>
    </Box>
```

The `& img` property don't work anymore, so we need to make sure all these replacement is strictly equal to the previous icon. We did a lot of effort the make the UI perfect, only then we can get a beautiful app.